services/web/src/modals/Organization.js

```diff
@@ -1,18 +1,41 @@
 import React from 'react';
-import { Modal } from 'semantic';
+import { Modal, Form } from 'semantic';
 import { modal } from 'helpers';
+import { Menu } from 'semantic-ui-react';
+import { Layout } from 'components';
 
 @modal
 export default class OrganizationSelector extends React.Component {
-
   render() {
     return (
       <React.Fragment>
-        <Modal.Header>
-          Select Account
-        </Modal.Header>
+        <Modal.Header>Select Organization</Modal.Header>
         <Modal.Content>
-          Choose account here
+          <Layout style={{ height: '300px' }}>
```

Comment:
maybe we don't need this? .modal .content should probably have a min height we can just do that instead?